packages/shared/styles/components/SfGallery.scss

```diff
@@ -121,3 +121,14 @@ $gallery__stage-cursor: zoom-in !default;
     }
   }
 }
+
+.sf-gallery__zoom {
+  z-index: 1000;
+  transition: 0.3s;
```

Comment:
```suggestion
```
Now it's doesn't work :) with property do you transform?

Reply:
I used the transition for the movement of the image when the user scrolls, the image is put back under the main one, it's a cool effect. 😬

Reply:
@juanfrantomaswac ok, then please set the right property for transition because `all` is not a good idea :)